apps/admin-gui/src/app/users/pages/user-detail-page/user-dashboard/user-dashboard.component.ts

```diff
@@ -25,7 +25,7 @@ export class UserDashboardComponent implements OnInit {
   roles: { [key: string]: { [key: string]: Array<number> } } = {};
   userProfileUrl = '';
   roleNames: string[];
-  isOnlySelfRole = false;
+  isOnlySelfOrMembershipRole = false;
```

Comment:
I think we should also include the role `SPONSORSHIP`. Also choose more suitable name for this variable.